src/main/fc/fc_rc.c

```diff
@@ -259,9 +262,18 @@ FAST_CODE uint8_t processRcInterpolation(void)
 }
 
 #ifdef USE_RC_SMOOTHING_FILTER
-uint8_t calcRcSmoothingCutoff(float avgRxFrameRate, float filterBaseline)
+int calcRcSmoothingCutoff(float avgRxFrameTimeMs, bool pt1)
 {
-    return lrintf(filterBaseline / (avgRxFrameRate / RC_SMOOTHING_FILTER_AUTO_MS));
+    if (avgRxFrameTimeMs > 0) {
+        float cutoff = (1 / (avgRxFrameTimeMs / 1000)) / 2; // calculate the nyquist frequency
+        cutoff = cutoff * 0.90f;  // Use 90% of the calculated nyquist frequency
+        if (pt1) {
+            cutoff = sq(cutoff) / 80; // convert to a cutoff for pt1 that has similar characteristics
```

Comment:
Please extract 80 to a constant

Reply:
Any suggestions on the constant name?  `BIQUAD_TO_PT1_CONVERSION_FACTOR` ?  I'm not sure it has any more technical name.

Reply:
We select cutoffs in such a way that attenuation at this frequency is equal across all filter types (pt1, biquad).

I would call it something like RC smoothing identity frequency, nothing else comes to mind